backend/cmd/generate.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"github.com/go-jet/jet/v2/generator/postgres"
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	err := postgres.GenerateDSN(
```

Comment:
I really dont like how this introduces a circular dependency. Can we avoid using codegen like this? Maybe move the source of truth to protobuf and generate the sql from that or something.

Reply:
what is the difference between this and what hasura needs to generate its metadata?

Reply:
fine maybe hasura is just bad

Reply:
Idk how you're supposed to declarative manage your database schema, I guess I'll look into that more